test/utils.js

```diff
@@ -1,6 +1,6 @@
 import mapboxgl from 'mapbox-gl';
 
-export const accessToken = 'pk.eyJ1IjoibWFwYm94IiwiYSI6IlhHVkZmaW8ifQ.hAMX5hSW-QnTeRCMAy9A8Q';
+export const accessToken = 'pk.eyJ1IjoiamZpcmUiLCJhIjoiZTFlNmQ3N2MzYmM2YzVjMzhkOTM2NTRhYzNiNGZiNGYifQ.1W47kmoEUpTJa3YIFefxUQ';
```

Comment:
Lets not have keys in the code. We can use env vars to help the tests run in circle if we need too.